### DISCUSSION TITLE
Rustdesk Server

### DISCUSSION THREAD

**Discussion by ghost** [runtipi/runtipi-appstore#2403] [Requests]
### Description

An open-source remote desktop host, with a web dashboard setup to work out of the box.

### Official Website

https://rustdesk.com/

### Documentation link

https://github.com/rustdesk/rustdesk?tab=readme-ov-file#how-to-build-on-linux

### Last application release & date

1.2.3 (oct, 13 2023)

### Application license

AGPL 3.0

### Source code repository link

https://github.com/rustdesk/rustdesk

### Docker image link

https://hub.docker.com/r/rustdesk/rustdesk-server

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by jasonpearce** [comment]
Are these duplicates? 

- https://github.com/runtipi/runtipi-appstore/discussions/2213
- https://github.com/runtipi/runtipi-appstore/discussions/2403

**Reply by JigSawFr** [reply]
Thanks for reporting !
